db/migrate/20231227085329_create_comments.rb

```diff
@@ -0,0 +1,12 @@
+# CreateComments migration
+class CreateComments < ActiveRecord::Migration[7.1]
+  def change
+    create_table :comments do |t|
+      t.references :user, null: false, foreign_key: true
+      t.references :post, null: false, foreign_key: true
```

Comment:
- [ ] There is an error with your migration files setup. You are referencing `posts` here, but with the file arrangement, the posts table has not been created yet, leading to an error. The arrangement of your migration files should be such that the users table is created first, followed by the posts table, then the comments and likes, but you are creating the posts after creating comments, so the comments table would not be migrated correctly, because comment depends on the posts table.

  ![image](https://github.com/DevKaranJ/Blog/assets/36542986/e5b3222a-7e23-461d-8a84-0e2c5a53edc1)